src/main/java/sirius/biz/storage/layer2/jdbc/SQLBlob.java

```diff
@@ -239,13 +245,18 @@ protected void beforeSave() {
             createdOrRenamed = true;
         }
 
+        if (isNew() || isChanged(CONVERTIBILITY_CHANGED)) {
```

Comment:
Will this not result in the field being set back to true when it is set to false via `mango.update`?